platform_versioned_docs/version-24.1.0/enterprise/release_notes/enterprise_latest.mdx

```diff
@@ -29,7 +26,7 @@ Three additional user roles have been introduced alongside Data Studios. These r
 
 ## Managed identities
 
-Seqera managed identities resolve the limitations of SSH access by enabling granular access control to HPC clusters for individual workspace users. 
+Seqera managed identities resolve the limitations of SSH access by enabling granular access control to HPC clusters for individual workspace users.
```

Comment:
What limitations? I assume this is:
- Anonymization of identity when many folks share the same Service Account key?
- Too broad permissions associated with the SA Key in order to have necessary filesystem permissions to interact with all necessary folder?
- Something else?

Reply:
@mbosio85, do you have additional content for this section? Thank you!

Reply:
> Anonymization of identity when many folks share the same Service Account key
It enables granular access control for individual workspace users, where before all users used the same SSH key which was one identity on the HPC side